NEWS

```diff
@@ -1,6 +1,43 @@
 This file lists noteworthy changes in SaunaFS.
 
+ * SaunaFS (4.7.0) (2024-12-18)
+ - fix(mount): Remove helgrind warning
+ - test: Reduce the helgrind mount suppressions
+ - fix(mount): Reduce data race conditions in mount
+ - test(all): Add new basic helgrind test
+ - fix(chunkserver): Replace pthread_mutex_t by std in jobpool (#252)
+ - chore: Change how PACKAGE_VERSION string is build (#232)
+ - fix(pcqueue): Remove Helgrind warnings (#250)
+ - test: Move pread to MachineTests (#249)
+ - refactor: Minor changes in Windows Client testing (#248)
+ - docs: Fix wrong default value in sfschunkserver.cfg (#246)
+ - refactor(common): Replace raw C-based pcqueue (#247)
+ - feat(chunkserver): Stat chunks at scan optionally (#245)
+ - doc(master): Add man docs for caseinsensitive (#243)
+ - chore: Changelog for v4.6.0 (#226)
```

Comment:
Is it expected to be here this change?

Reply:
No, it is not expected. Also code maintenance stuff (e.g chores) are not needed for the changelog